tmux.conf

```diff
@@ -35,5 +35,8 @@ bind-key C-b send-prefix
 # prefix-2 -> forward-incremental-history-search
 bind-key C-s send-prefix -2
 
+# don't suspend-client
+tmux unbind C-z
```

Comment:
`tmux` will cause an error here, I believe.

Also, I'm in favor of using the full command `unbind-key` to be extra clear, rather than the short form `unbind`.

So all total, I think this could be:

```
unbind-key C-z
```